benchmark/atoi.cpp

```diff
@@ -54,7 +54,7 @@ uint32_t calculateBase10(zoo::swar::SWAR<8, uint64_t> convertedToIntegers) noexc
     return uint32_t(by10001base2to32.value() >> 32);
 }
 
-uint64_t calculateBase10(zoo::swar::SWAR<8, __uint128_t> convertedToIntegers) noexcept {
+uint64_t calculateBase10_128(zoo::swar::SWAR<8, __uint128_t> convertedToIntegers) noexcept {
```

Comment:
The naming`calculateBase10` vs `calculateBase10_128` is possibly confusing from an API perspective given they both ostensibly take in a 128bit SWAR. 

It wouldn't be obvious as a user of these functions which one to choose.

Is there also a world in which we would make this function a part of an API and the users can choose which implementation to use from a template? 

```cpp
calculateBase10<UseMaxNativeRegisterSize>(...); // default 
calculateBase10<Emulate128Bits>(...);
```

Reply:
These are not, at the moment, user-facing APIs.  This is changing.